noxfile.py

```diff
@@ -90,6 +91,14 @@ def unit(session):
     default(session)
 
 
+@nox.session(python="3.8")
+def pytype(session):
```

Comment:
Can we add this to `nox.options.sessions` so that it runs in the `Kokoro` build?

Reply:
Done. From [Kokoro log](https://source.cloud.google.com/results/invocations/fa869575-8829-4345-8cbc-ed10b88abed7/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery%2Fpresubmit%2Fpresubmit/log):
```
...
nox > Session blacken was successful.
nox > Running session pytype
nox > Creating virtual environment (virtualenv) using python3.8 in .nox/pytype
nox > python -m pip install pytype==2021.4.9
nox > pytype
Computing dependencies
Analyzing 40 sources with 7 local dependencies
ninja: Entering directory `.pytype'
[1/69] check google.cloud.bigquery.encryption_configuration
...
```